de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/search/DataSearchPresenterImpl.java

```diff
@@ -364,6 +367,27 @@ public void onSuccess(Tag result) {
         });
     }
 
+    @Override
+    public void searchCollaborators(String searchTerm, ReactCallback collaboratorCallback) {
+        collaboratorsServiceFacade.searchCollaborators(searchTerm, new AsyncCallback<List<Subject>>() {
+            @Override
+            public void onFailure(Throwable caught) {
+                ErrorHandler.post(caught);
+            }
+
+            @Override
+            public void onSuccess(List<Subject> result) {
+                if (collaboratorCallback != null) {
+                    Splittable data = StringQuoter.createIndexed();
+                    result.forEach(subject -> {
+                        Splittable splSubject = AutoBeanCodex.encode(AutoBeanUtils.getAutoBean(subject));
+                        splSubject.assign(data, data.size());
+                    });
+                    collaboratorCallback.onSuccess(data);
```

Comment:
Ah, I remember you making that suggestion and I completely spaced on it here.  I think it will actually only be a couple lines to change, so I'll put it in this PR so I don't forget 😉